src/app/shared/entry/info-tab-topic/edit-topic/edit-topic-dialog.component.html

```diff
@@ -57,22 +58,48 @@ <h1 mat-dialog-title>Select a Topic</h1>
           </div>
         </mat-radio-button>
       </mat-card>
-      <mat-card>
-        <mat-radio-button [value]="TopicSelectionEnum.AI" data-cy="topic-ai-selection-button">
-          <div>
-            <span class="bubble mr-2"><img class="site-icons-small" src="../../../assets/svg/ai-icon.svg" alt="AI generation icon" /></span>
-            <strong>AI</strong>
+      <mat-card *ngIf="entry.topicAI">
+        <mat-radio-button [value]="TopicSelectionEnum.AI" data-cy="topic-ai-selection-button" class="w-100">
+          <div fxLayoutAlign="space-between">
+            <div>
+              <span class="bubble mr-2"
+                ><img class="site-icons-small" src="../../../assets/svg/ai-icon.svg" alt="AI generation icon"
+              /></span>
+              <strong>AI</strong>
+            </div>
+            <span
+              *ngIf="entry.approvedAITopic"
+              class="bubble approved"
+              matTooltip="This AI topic was reviewed and selected by the {{ entryTypeMetadata.term }} owner"
```

Comment:
Nit, I'd prefer it say `selected by a...`, because an entry can have multiple owners. Fortunately, all of our terms start with a consonant, so you wouldn't have to figure out if it should be `a` or `an`. :)

I could see this being overly semantic, so no problem if you'd rather keep it.